VAMobile/src/components/WaygateWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { AlertBox, Box } from 'components'
+import React, { FC, ReactElement } from 'react'
+
+import { Waygate, WaygateToggleType, waygateEnabled } from 'utils/remoteConfig'
+import { useTheme } from 'utils/hooks'
+
+export type WaygateWrapperProps = {
+  /** the waygate name to check for */
+  waygate: WaygateToggleType
+}
+
+/**A common component for the react native switch component*/
+export const WaygateWrapper: FC<WaygateWrapperProps> = ({ children, waygate }) => {
+  const theme = useTheme()
+
+  const waygateAlertBox = (wg: Waygate): ReactElement => {
+    return (
+      <Box mb={theme.dimensions.condensedMarginBetween}>
+        <AlertBox border="warning" title={wg.errorMsgTitle} text={wg.errorMsgBody} />
```

Comment:
I think i'll need to conditionalize the border type in the future to be warning/error.